server/routes.js

```diff
@@ -1,10 +1,38 @@
-const express = require('express');
+const router = require('express').Router();
+// const log = require('loglevel');
+// const validateRequest = (req, res, next) => {
+//   next();
+// };
+const {
+  stakeholderGet,
+  stakeholderGetAll,
+  stakeholderGetUnlinked,
+  stakeholderUpdateLink,
+  stakeholderPatch,
+  stakeholderPost,
+} = require('./handlers/stakeholderHandler');
+const { handlerWrapper } = require('./utils/utils');
 
-const router = express.Router();
-const { stakeholderGet } = require('./handlers/stakeholderHandler');
+router
+  .route('/links/:stakeholder_id/:acctStakeholder_id')
+  .get(handlerWrapper(stakeholderGetUnlinked))
+  .patch(handlerWrapper(stakeholderUpdateLink));
 
-const { handlerWrapper } = require('./utils/utils');
+router
+  .route('/links/:stakeholder_id')
+  .get(handlerWrapper(stakeholderGetUnlinked))
+  .patch(handlerWrapper(stakeholderUpdateLink));
```

Comment:
Creating and deleting a link makes sense to execute against the /links route, but shouldn't this be a resource create and delete (POST and DELETE) rather than a patch?  A link is just a resource that joins two other resources.

Perhaps the resource should be called 'relationships' instead of links as well?  (we call them 'entity relationships' in the legacy system)